packages/frontend/src/utils/wallet.js

```diff
@@ -616,7 +636,7 @@ export default class Wallet {
     await contract.create_account_and_claim(
       {
         new_account_id: accountId,
-        new_public_key: publicKey.toString().replace("ed25519:", ""),
+        new_public_key: publicKey.toString()//.replace("ed25519:", ""),
```

Comment:
Please remove instead of comment if not relevant anymore

Reply:
Oh sorry I missed this - I'm going to keep it, it was just breaking something for my trial contract but this has been fixed.